<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MutiGameForm</class>
 <widget class="QWidget" name="MutiGameForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="time_label">
         <property name="text">
          <string>0秒</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="time_progressBar">
         <property name="value">
          <number>24</number>
         </property>
         <property name="format">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="canvas_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>700</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>700</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="start_pushButton">
       <property name="text">
        <string>开始游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="return_pushButton">
       <property name="text">
        <string>返回主菜单</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="blueScore_label">
       <property name="text">
        <string>BlueScore: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="greenScore_label">
       <property name="text">
        <string>GreenScore: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="hint_label">
       <property name="text">
        <string>HintTime: 0s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="blueFreeze_label">
       <property name="text">
        <string>BlueFreeze: 0s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="blueDizzy_label">
       <property name="text">
        <string>BlueDizzy: 0s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="greenFreeze_label">
       <property name="text">
        <string>GreenFreeze: 0s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="greenDizzy_label">
       <property name="text">
        <string>GreenDizzy: 0s</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stop_pushButton">
       <property name="text">
        <string>暂停</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="continue_pushButton">
       <property name="text">
        <string>继续</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_pushButton">
       <property name="text">
        <string>存储</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_pushButton">
       <property name="text">
        <string>加载</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
